components/masking_functions/src/masking_functions/query_cache.cpp

```diff
@@ -0,0 +1,133 @@
+/* Copyright (c) 2024 Percona LLC and/or its affiliates. All rights reserved.
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; version 2 of the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software Foundation,
+   51 Franklin Street, Suite 500, Boston, MA 02110-1335 USA */
+
+#include "masking_functions/query_cache.hpp"
+
+#include "masking_functions/command_service_tuple.hpp"
+#include "masking_functions/primitive_singleton.hpp"
+#include "masking_functions/query_builder.hpp"
+
+#include <algorithm>
+
+namespace masking_functions {
+namespace {
```

Comment:
You are not declaring any data in the anonymous namespace, just typedefs. So, do not really see any benefit of using it here.

Reply:
It now has some additional items in there, not only typedefs.